### PR TITLE
Lib Refactor

### PR DESCRIPTION
**Pull request by varun13169**
1) Add Grid NavBar
2) Badge border dynamic to the content
3) Add Card prod horizontal and vertical css

### CODE REVIEW

File: library/navbar/navbar.css [hunk 1]
```diff
@@ -83,4 +83,114 @@
...
+    margin-right: 3rem;
+}
+
+.dui-nav-sch-act__auth-actions>*:not(:last-child) {
+    margin-right: 2rem;
+}
+
+.dui-nav-sch-act__signup-btn {}
+
+.dui-nav-sch-act__login-btn {}
```
Comment (zoef-yt) [inline library/navbar/navbar.css:127]: remove unused classes. If they are for future reference do mention it in with the commented line

**Review by zoef-yt — APPROVED** [review]
Great work. Looks good to merge.

File: library/card/card.css [hunk 2]
```diff
@@ -203,4 +203,169 @@
...
+
+.dui-card-prod-hzntl {
+    width: fit-content;
+    height: auto;
+
+    background: var(--dui-light-theme-bg);
+    display: grid;
+
+    grid-auto-columns: 4fr 4fr 3fr 3fr 3fr 3fr;
+    grid-template-areas:
```
Comment (humzahasan) [inline library/card/card.css:218]: Suggestion  - 

grid-auto-columns: 2fr 1fr;

grid-template-areas: 
 'cardImg  cardInfo'
 'cardImg  cardDesc '
 'cardImg  cardActions ';
 
 Won't this help you ?

**Review by humzahasan — APPROVED** [review]
Feedback : You can check on my suggestion helps to get the similar result.

Code looks good to merge!

**Pull request merged**